machine/accton/accton_as_28t_ops/u-boot/platform-as_28t_ops.patch

```diff
@@ -0,0 +1,2665 @@
++	__u_boot_cmd_end = .;
++
++	. = ALIGN(4);
++
++	__image_copy_end = .;
++
++	.rel.dyn : {
++		__rel_dyn_start = .;
++		*(.rel*)
++		__rel_dyn_end = .;
++	}
++
++	.dynsym : {
++		__dynsym_start = .;
++		*(.dynsym)
++	}
++
++	.bss __rel_dyn_start (OVERLAY) : {
++		__bss_start = .;
++		*(.bss)
++		 . = ALIGN(4);
++		__bss_end__ = .;
++        _end = .;
++	}
++
++	.irqstack : {
++		__irqstack_end = .;
++		. = . + 0x2008;
++		__irqstack_start = .;
++	}
++
++	/DISCARD/ : { *(.dynstr*) }
++	/DISCARD/ : { *(.dynamic*) }
++	/DISCARD/ : { *(.plt*) }
++	/DISCARD/ : { *(.interp*) }
++	/DISCARD/ : { *(.gnu*) }
++}
+diff --git a/boards.cfg b/boards.cfg
+index 368f3c4..3540c9a 100644
+--- a/boards.cfg
++++ b/boards.cfg
+@@ -308,6 +308,7 @@ hurricane2_nand              arm         armv7       bcm95615x           broadco
+ hurricane2_nor              arm         armv7        bcm95615x           broadcom       hurricane2        hurricane2:L2C_AS_RAM,RUN_DDR_SHMOO2,CUSTOM_LINKER_SCRIPT,NOR_IPROC_BOOT
+ hurricane2_noshmoo           arm         armv7       bcm95615x           broadcom       hurricane2        hurricane2:CUSTOM_LINKER_SCRIPT
+ hurricane2_rom_emul          arm         armv7       bcm95333x           broadcom       hurricane2        hurricane2_rom_emul:L2C_AS_RAM,NO_CODE_RELOC
++AS_28T_OPS                   arm         armv7       as_28t_ops           accton        hurricane2        AS_28T_OPS:ONIE_PLATFORM_REV=0
+ katana2                       arm         armv7       bcm95645x           broadcom       katana2           katana2:RUN_DDR_SHMOO2,IPROC_DDR_ECC 
+ katana2_nand                  arm         armv7       bcm95645x           broadcom       katana2           katana2:RUN_DDR_SHMOO2,IPROC_DDR_ECC,NAND_IPROC_BOOT
+ origen			     arm	 armv7	     origen		 samsung	exynos
+diff --git a/broadcom_boards.mk b/broadcom_boards.mk
+index 60b2ecf..ba75cdf 100644
+--- a/broadcom_boards.mk
++++ b/broadcom_boards.mk
+@@ -151,6 +151,10 @@ hurricane2_rom_emul_config: $(obj)hurricane2_rom_emul_config
+ $(obj)hurricane2_rom_emul_config: $(TOPDIR)/include/configs/hurricane2_rom_emul.h 
+ 	@$(MKCONFIG) -A $(@F:_config=); touch $@
+ 
++AS_28T_OPS_config: $(obj)AS_28T_OPS_config
++$(obj)AS_28T_OPS_config: $(TOPDIR)/include/configs/AS_28T_OPS.h
++	@$(MKCONFIG) -A $(@F:_config=); touch $@
++
+ katana2_config: $(obj)katana2_config 
+ $(obj)katana2_config: $(TOPDIR)/include/configs/katana2.h  
+ 	@$(MKCONFIG) -A $(@F:_config=); touch $@
+diff --git a/common/cmd_sys_eeprom.c b/common/cmd_sys_eeprom.c
+index ab02104..7839324 100644
+--- a/common/cmd_sys_eeprom.c
++++ b/common/cmd_sys_eeprom.c
+@@ -8,6 +8,8 @@
+ #include <common.h>
+ #include <command.h>
+ #include <i2c.h>
++#include <spi_flash.h>
++#include <environment.h>
+ #include <linux/ctype.h>
+ 
+ #include "sys_eeprom.h"
+@@ -926,6 +928,82 @@ int write_sys_eeprom(void *eeprom, int len)
+ 
+ #endif /* CONFIG_SYS_FLASH_HWINFO_ADDR && CONFIG_SYS_FLASH_HWINFO_SECT_SIZE */
+ 
++#ifdef CONFIG_SYS_EEPROM_USE_COMMON_SPI_IO
```

Comment:
This SPI IO patch looks generally like a good thing.  It appears to be the same as the AS4510_52T code.

Can this be factored out into the common sys_eeprom patch?  Can we make it part of patches/u-boot/common/feature-sys-eeprom-tlv-common.patch , like `CONFIG_SYS_EEPROM_USE_COMMON_FLASH_IO` and `CONFIG_SYS_EEPROM_USE_COMMON_I2C_IO` ?